File: JavaScript/Car.js
export default class Car {
  // Constructor que asigna los valores a las llantas
  constructor(A, B, C, D) {
    this.rightFront = A;
    this.leftFront = B;
    this.rightRear = C;
    this.leftRear = D;
  }

  // Método para entrar a pits
  Pits() {
    console.log("The car is entering to pits and the tyres will be changed");
  }
}

File: C#/Classes/Car.cs
//esta clase representa un carro con 4 llantas
public class Car
{
    //Aca defino los parametros del carro (sus llantas)
    public string rightFront;
    public string leftFront;
    public string rightRear;
    public string leftRear;
    //Este es el metodo constructor que le asigna valores a los parametros
    public Car(string A, string B, string C, string D)
    {
        rightFront = A;
        leftFront = B;
        rightRear = C;
        leftRear = D;
    }

    //Aca defino el metodo de pits que sera necesario cuando las llantas esten desgastadas
    public void Pits()
    {
        Console.WriteLine("The car is entering to pits and the tyres will be change");
    }

}